public/js/action-list.js

```diff
@@ -82,14 +82,26 @@
                 $list.find('.list-item.active').removeClass('active');
                 $item.addClass('active');
             }
+
+            // For items that do not have a bottom status bar like Downtimes, Comments...
+            if (! $list.closest('.container').find('.footer').length) {
+                $list.closest('.container').append('<div class="footer automatically-added"></div>');
```

Comment:
Don't add a class if you don't require it for styling. If you need to flag a particular DOM element, use data attributes instead. (e.g. `<div class="footer" data-action-list-automatically-added></div>`)